test/s3select_test.cpp

```diff
@@ -524,6 +524,16 @@ void generate_csv_escape(std::string& out, size_t size) {
   out = ss.str();
 }
 
+void generate_columns_csv(std::string& out, size_t size) {
+  std::stringstream ss;
+  auto r = [](){return 1;};
```

Comment:
why return 1?

Reply:
Unlike random values, outcome is predictable.

Reply:
an object which contain only single value for columns and rows, may cause a predicate to return false for all rows/columns or true for all rows/columns. 
it has the same impact of single row/column object

its better to build an object with fixed and different values (not changed between runs)
such as 
 1,2,3,4,5...
 2,4,6,....
or any other static object structure (could contain a mix of types)